bin/runtime/src/lib.rs

```diff
@@ -1000,10 +964,8 @@ construct_runtime!(
         Identity: pallet_identity = 20,
         CommitteeManagement: pallet_committee_management = 21,
         Proxy: pallet_proxy = 22,
-        FeatureControl: pallet_feature_control = 23,
-        VkStorage: pallet_vk_storage = 24,
-        SafeMode: pallet_safe_mode = 25,
-        TxPause: pallet_tx_pause = 26,
+        SafeMode: pallet_safe_mode = 23,
+        TxPause: pallet_tx_pause = 24,
```

Comment:
I know @kostekIV already asked this, but what's the situation here? Do we have any pallet number >=23 on testet/mainnet?

Reply:
We do have https://github.com/Cardinal-Cryptography/aleph-node/blob/r-13.3/bin/runtime/src/lib.rs#L940-L966 
```
        BabyLiminal: pallet_baby_liminal = 41,
        Operations: pallet_operations = 255,
```
Although it does not matter since those are not 23 or 24? Also, those pallets do not keep any storages